> What was the used cmake configuration command? Can you paste that along with the error from ctest

@aaronj0 



Used CMake configure command:

```bash
cmake -DBUILD_SHARED_LIBS=ON -DCPPINTEROP_USE_CLING=ON -DCPPINTEROP_USE_REPL=Off -DCling_DIR=$LLVM_DIR/build/tools/cling -DLLVM_DIR=$LLVM_DIR/build/lib/cmake/llvm -DClang_DIR=$LLVM_DIR/build/lib/cmake/clang -DCMAKE_INSTALL_PREFIX=$CPPINTEROP_DIR ..
```

CTest/check target error before the fix:

```text
CMake Error: Invalid value used with --build-config
```

Full tail from the failing run:

```text
[100%] Built target CppInterOpUnitTests
CMake Error: Invalid value used with --build-config
gmake[3]: *** [unittests/CMakeFiles/check-cppinterop.dir/build.make:70: unittests/CMakeFiles/check-cppinterop] Error 1
gmake[2]: *** [CMakeFiles/Makefile2:659: unittests/CMakeFiles/check-cppinterop.dir/all] Error 2
gmake[1]: *** [CMakeFiles/Makefile2:666: unittests/CMakeFiles/check-cppinterop.dir/rule] Error 2
gmake: *** [Makefile:351: check-cppinterop] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,6 +24,13 @@ endif()
 if( CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR )
   project(CppInterOp)
 
+  if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
+    message(WARNING "No build type selected. Defaulted CMAKE_BUILD_TYPE=Release.\n"
+      "Valid options are: None, Debug, Release, RelWithDebInfo, MinSizeRel.")
+    set(CMAKE_BUILD_TYPE Release CACHE STRING
+      "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel" FORCE)
+  endif()
+
   # LLVM/Clang/Cling default paths
   if (DEFINED LLVM_DIR)
     if (NOT DEFINED Clang_DIR)

